apps/src/p5lab/spritelab/SpriteLab.js

```diff
@@ -160,4 +160,15 @@ export default class SpriteLab extends P5Lab {
   getReinfFeedbackMsg(isFinalFreePlayLevel) {
     return isFinalFreePlayLevel ? null : this.getMsg().reinfFeedbackMsg();
   }
+
+  runValidationCode() {
+    // Skip validation code in 'editBlocks' mode (i.e., a levelbuilder is
+    // editing start blocks for the level).
+    if (this.level.editBlocks) {
```

Comment:
can levelbuilders not see the results of their code on run now? I'm not sure what should happen here since we want it to save immediately, but I wonder if they will be concerned if they can't see what the code does.

Reply:
ahhh, that's a good question. @mikeharv do you have thoughts here? feel free to redirect me to someone on the curriculum team if you don't have an opinion!

Reply:
I think this is a reasonable side effect. To see the results of their updated start code, they'd just need to go back to the level view and reset their version history.

The alternative that comes to mind is that we could always provide a Finish button and use that to save. We would still want to disable validation code in that case if possible. I'm not sure how feasible this idea is. I'm currently trying to figure out if the "Free Play" option actually does anything in Sprite Lab levels. It seems to be ignored if validation_code isn't empty.

Reply:
Considering this is such a small change and it only impacts levelbuilders working with Sprite Lab, I'd feel fine about merging this change and asking the curriculum team to see how it feels. If they are missing the experience of seeing what their code does, we could revert or do a follow-up. Also FWIW, Sprite Lab does still have a live preview so it's only the event/behavior code that they wouldn't see right away.

Reply:
good point -- this ended up being an easy change that we can tweak later (now that i understand how validation/feedback works in this area). once this change is on levelbuilder, i'll let Amy BW know -- are there any other CSF folks i should notify?

Reply:
Katie Frank and Emma Wingreen